### PR TITLE
bundler 0.0.3

### PR DESCRIPTION
**Pull request by chrisfuller**
TagCheck

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [X] __I will not move an already shared step version's tag to another commit__
- [X] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [X] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [X] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [X] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

### CODE REVIEW

File: steps/bundler/0.0.3/step.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+support_url: https://github.com/chrisfuller/bitrise-step-bundler/issues
+published_at: 2019-09-08T16:41:15.363368-07:00
+source:
+  git: https://github.com/chrisfuller/bitrise-step-bundler.git
+  commit: 5941e841a021216b90c13e32f1b38bfdc7ac6691
+host_os_tags:
+- osx
+- ubuntu
+type_tags:
+- utility
```
Comment (godrei) [inline steps/bundler/0.0.3/step.yml]: pls remove `utility` type_tag in the next release, as this step handles dependencies. `utility` is used if no other category matches the step.

File: steps/bundler/0.0.3/step.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+is_always_run: false
+is_skippable: false
+inputs:
+- bundle_install_jobs: "4"
+  opts:
+    description: |
+      The maximum number of parallel download and install jobs.
+    is_expand: true
+    is_required: true
+    title: --jobs
```
Comment (godrei) [inline steps/bundler/0.0.3/step.yml]: The underlying flag is not the best as step input title, as may not all of the Bitrise users know what is it for, `The maximum number of parallel jobs` sounds better, we usually mention the underlying flag in the input's description.

**Comment by godrei** [comment]
@chrisfuller thanks for the new step version, i approve your PR, but pls fix the mentioned things in the next step release.

File: steps/bundler/0.0.3/step.yml (hunk 1 above)
Comment (chrisfuller) [reply steps/bundler/0.0.3/step.yml]: Removed in `v0.0.4` 👍

File: steps/bundler/0.0.3/step.yml (hunk 2 above)
Comment (chrisfuller) [reply steps/bundler/0.0.3/step.yml]: Fixed in `v0.0.4` 👍

**Pull request merged**